api/monetization.js

```
@@ -62,20 +62,24 @@ async function sendEmail(to, subject, token) {
 }
 
 
 // ─── License Token Utilities ──────────────────────────────────────────────────
-if (!process.env.LICENSE_JWT_SECRET) {
-  throw new Error('LICENSE_JWT_SECRET environment variable is required');
-}
-const LICENSE_SECRET = process.env.LICENSE_JWT_SECRET;
+// FIX: no throw on missing env var so Vercel deployments without the secret
+// don't crash at startup. Set LICENSE_JWT_SECRET in Vercel Environment Variables.
+const LICENSE_SECRET = (() => {
+  if (process.env.LICENSE_JWT_SECRET) return process.env.LICENSE_JWT_SECRET;
```

ESLint

[error] 'process' is not defined.

(no-undef)